ProcessLib/TimeLoop.cpp

```diff
@@ -720,6 +725,14 @@ TimeLoop::solveCoupledEquationSystemsByStaggeredScheme(
             timestep_id, t);
     }
 
+    if (_chemical_system != nullptr)
```

Comment:
Suggestion: to move this into the coupling loop.

Reply:
Add a comment on coupling scheme for reactive solute transport.